clearwater-sip-stress-coreonly.root/usr/share/clearwater/sip-stress/callee_redirection.xml

```diff
@@ -0,0 +1,321 @@
+<?xml version="1.0" encoding="ISO-8859-1" ?>
+<!DOCTYPE scenario SYSTEM "sipp.dtd">
+
+<scenario name="Clearwater callee">
+  <recv request="INVITE" rrs="true" crlf="true">
+    <action>
+      <!-- Save off our identity for later use -->
+      <ereg regexp=".*<sip:(.*)@(.*)>.*" search_in="hdr" header="To:" check_it="true" assign_to="_,dn,home_domain" />
+    </action>
+  </recv>
+
+  <nop>
+    <action>
+      <!-- Save off the per-transaction headers, for use on the eventual 180 Ringing and 200 OK -->
+      <assignstr assign_to="invite_cseq" value="[last_Cseq:]" />
+      <assignstr assign_to="invite_via" value="[last_Via:]" />
+    </action>
+  </nop>
+
+  <!-- Here we send two 302 messages to simulate nested redirection and to
+       increase load on the BGCF -->
+  <send>
+    <![CDATA[
+      SIP/2.0 302 Moved Temporarily 
+      [last_Call-ID:]
+      [last_CSeq:]
+      [last_From:]
+      [last_To:];tag=[pid]SIPpTag01[call_number]
+      [last_Via:]
+      Server: SIP/2.0
+      Content-Length: [len]
+      [last_P-Charging-Vector:]
+      P-Asserted-Identity: sip:[$dn]@[$home_domain]
+      Contact: <sip:8888000002424@ame22.clearwater.test>;q=0.2,<sip:8888000001313@ame22.clearwater.test>;q=0.1,<sip:8888000003535@ame22.clearwater.test>;q=0.3
```

Comment:
Didn't spot this on initial review, but have seen it while investigating an issue - this hardcodes ame22.clearwater.test so won't work against any other deployment. Let's discuss what to do here.